### DISCUSSION TITLE
Possibility to replace long-running pods / lifecycle policies

### DISCUSSION THREAD

**Discussion by FoxDawg** [kedacore/keda#5318] [Q&A / Need Help] [answered]
Hi there,

is there any possibility to create a trigger that automatically recycles long-running instances?
Like, for example, if I want my pods to be recreated after 2days to keep the cluster fresh and healthy - is there any such thing with KEDA?
Thanks!

**Comment by zroubalik** [comment, marked as answer]
Hi, KEDA doesn't support this at the moment.

**Reply by FoxDawg** [reply]
Thanks!
Any other chance then to tell KEDA to take down the _oldest_ instances in case of downscaling events? That would be a decent compromise in our environment.

**Reply by zroubalik** [reply]
The actuall scaling is handled by Kubernetes HPAs, KEDA doesn't have any saying in it.

**Reply by FoxDawg** [reply]
Good point. Maybe a cron job in Kubernetes might be a way to go. Thx!
